cephprimarystorage/cephprimarystorage/imagestore.py

```
@@ -63,6 +70,31 @@ def _get_image_reference(self, path):
+        result = jsonobject.loads(output.splitlines()[-1])
+
+        _, image = self._get_id_name_from_install(cmd.volumePath)
+        rsp = StorageBackupRsp()
+        rsp.installPath = os.path.join(cmd.uploadDir, result.backupName)
+        rsp.mode = result.mode
+        rsp.size = int(result.size)
+        return jsonobject.dumps(rsp)
+
+    def cancel_storage_backup(self, cmd):
+        self._check_zstore_cli()
+        sshData = cmd.user + '@' + cmd.host + ':' + cmd.uploadDir
```

Ruff (0.11.9)

Local variable `sshData` is assigned to but never used

Remove assignment to unused variable `sshData`

(F841)